redeploy.sh

```diff
@@ -32,6 +32,16 @@ docker rmi hubmap/portal-ui:latest
 
 echo 'starting...'
 docker-compose -f \$COMPOSE_CONFIG up -d
+
+# Need to restart the hubmap-auth container on STAGE to avoid
+# portal.stage.hubmapconsortium.org and portal-prod.stage.hubmapconsortium.org get swapped
+if [ "$TARGET" = "stage" ]; then
+    echo 'Restart the hubmap-auth container... for STAGE only'
```

Comment:
Is there any harm in doing this for every instance? I prefer that things be consistent.

Reply:
STAGE is less frequently being used by other developers and this `portal-prod.stage` is only available on STAGE. We don't have this issue on DEV/TEST/PROD because we are only running one portal-ui instance. So this is a unique use case on STAGE only.